doc/man3/PEM_read_bio_PrivateKey.pod

```diff
@@ -67,8 +67,14 @@ PEM_write_bio_PKCS7, PEM_write_PKCS7 - PEM routines
                                    char *kstr, int klen,
                                    pem_password_cb *cb, void *u);
 
+ EVP_PKEY *PEM_read_bio_PUBKEY_ex(BIO *bp, EVP_PKEY **x,
```

Comment:
Need to run doc-nits on this..
They are not in the NAME section

Reply:
Fixed